src/styles/bar.css

```diff
@@ -63,8 +70,21 @@
   text-indent: -9999em;
```

Comment:
I wonder if it's not a good case for using `$hidden` or `.coz-hidden` 
https://github.com/cozy/cozy-ui/blob/v3/stylus/ui-base/mixins.styl#L25